OrbitGl/DataManager.h

```diff
@@ -41,9 +43,11 @@ class DataManager final {
   void set_selected_process(std::shared_ptr<Process> process);
 
   [[nodiscard]] ProcessData* GetProcessByPid(int32_t process_id) const;
-  [[nodiscard]] const std::vector<ModuleData*>& GetModules(int32_t process_id) const;
-  [[nodiscard]] ModuleData* FindModuleByAddressStart(int32_t process_id,
-                                                     uint64_t address_start) const;
+  [[nodiscard]] ModuleData* GetModuleByPath(const std::string& path) const;
```

Comment:
This one can not. The module* that is returned here will be filled with symbols later. Loading symbols alters the module, hence it can't be const here

Reply:
Would it be feasible to move the symbol loading modifications to data manager (in a different PR)? At least I would add a comment or rename the method to `GetMutableModuleByPath`, or add it as an additional one.

Reply:
yes, that is a good idea. I added an additional `GetMutableModuleByPath`